### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

**Review by liuda-chuzdiuk — APPROVED** [review]

File: tests/test_main.py [hunk 1]
```diff
@@ -1 +1,31 @@
...
+):
+    mocked_get_exchange_rate_prediction.return_value = 9.4
+
+    assert cryptocurrency_action(10) == "Sell all your cryptocurrency"
+    assert cryptocurrency_action(15) == "Sell all your cryptocurrency"
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_if_difference_is_not_that_much(mocked_get_exchange_rate_prediction):
+    mocked_get_exchange_rate_prediction.return_value = 10
```
Comment (MasterpieceElbow) [inline tests/test_main.py:28]: Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 or 0.95 * value passed in `cryptocurrency_action`.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

File: tests/test_main.py [hunk 2]
```diff
@@ -1 +1,48 @@
...
+
+
+@pytest.mark.parametrize(
+    "exchange_rate,current_rate, expected",
+    [
+        (10.6, 10.1, "Do nothing"),
+        (10.6, 11.1, "Do nothing"),
+        (10.6, 10.7, "Do nothing"),
+        (10.6, 10, "Buy more cryptocurrency"),
+        (10.6, 11.2, "Sell all your cryptocurrency")
```
Comment (MasterpieceElbow) [inline tests/test_main.py:35]: Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 or 0.95 * value passed in `cryptocurrency_action`.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

**Review by IvanRamyk — APPROVED** [review]
Well done!
